src/lando/dockerflow/decorators.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import time
+from functools import wraps
+
+# Logging is not yet implemented
+request_logger = logging.getLogger("dockerflow.request_summary.logger")
```

Comment:
Curious where "dockerflow.request_summary.logger" came from? I think generally we do `logging.getLogger(__name__)`.

Reply:
It's similarly named in the original file, I was just being more verbose since it only applies to requests for Dockerflow.

https://github.com/mozilla-conduit/lando/blob/c17b6829e6851381390cf5ccaa3741320c7d4415/src/lando/ui/legacy/dockerflow.py#L10C1-L11C54

I removed the original line that follows the standard because it wasn't being used.